### DISCUSSION TITLE
Production use without docker?

### DISCUSSION THREAD

**Discussion by jaretclifton** [cookiecutter/cookiecutter-django#5314] [Q&A] [answered]
I'm brand new to cookiecutter-django and have been scouring every article and video I can find to try and answer this question. I don't want to use docker for production deployments. Is this a deal breaker? Is there an alternative method? I can't seem to find a solid answer. Thank you for the help!

**Comment by browniebroke** [comment, marked as answer]
No, it's not a deal breaker. You can choose to opt-out of Docker and will get a project without it. That means that you'll have to configure some components yourself that cookiecutter-django can't do for you: database, cache, reverse proxy, etc... 

If you look at the Heroku or Python anywhere docs, it should give you a good idea of what's specific to cookiecutter-django to start the app where you want to deploy it. Then the rest is mostly best practices to run a Python web app.

**Comment by foarsitter** [comment]
Search for 'host Django' and you will find a ton of information. 

When your are learning my advice would be to stick to the provided tools (docker in this case). Changing things without knowing what you are doing will cause a great headache.

**Comment by jaretclifton** [comment]
Much appreciated for the replies! I'm in a situtation where the organization I work at doesn't support Docker so I'm forced to either do things manually using cookiecutter as mentioned by @browniebroke or manually with Django from scratch. Was hoping to avoid the latter. I'm sure I'll have a ton of headaches fumbling my way through this, but the alternative seems like far more of a headache. Truly appreciate the insight!

**Reply by foarsitter** [reply]
Can you elaborate a little more why your company doesn't support docker? Since you have to install the django stack yourself it doesn't seem to me they support that either. If you have to do things manually anyway why not install docker yourself? It would be far less complicated then installing the rest manually.
